scripts/deploy_docs.sh

```diff
@@ -0,0 +1,27 @@
+DEPLOY_NIGHTLY_GITHUB_USER=leanprover-community-bot
+
+set -e
+set -x
+
+# By default, github actions overrides the credentials used to access any
+# github url so that it uses the github-actions[bot] user.  We want to access
+# github using a different username.
+git config --unset http.https://github.com/.extraheader
+
+git_hash="$(git log -1 --pretty=format:%h)"
+git clone https://github.com/leanprover-community/doc-gen.git
+cd doc-gen
+sed -i "s/rev = \"\S*\"/rev = \"$git_hash\"/" leanpkg.toml
+echo -e "builtin_path\npath ./src\npath ../src" > leanpkg.path
+git clone "https://$DEPLOY_NIGHTLY_GITHUB_USER:$DEPLOY_NIGHTLY_GITHUB_TOKEN@github.com/leanprover-community/mathlib_docs_test.git"
+rm -rf mathlib_docs/docs/*
+~/.elan/bin/elan override set leanprover-community/lean:nightly
```

Comment:
```suggestion
elan override set leanprover-community/lean:nightly
```

`elan` is in the `PATH`.